### ISSUE TITLE
Building docs locally is failling for me

### ISSUE THREAD

**Issue by svx** [plone/guillotina#660]
I am trying to build the docs locally, since I was not able to find docs about it, I tried it like this:

```shell
git clone git@github.com/plone/guillotina
cd guillotina
python3.7 -m venv .venv
source .venv/bin/activate
pip install guillotina
pip install -r docs-requirements.txt
```

After the installation is done

```shell
cd docs
vim Makefile # -> SPHINXBUILD   = ../.venv/bin/sphinx-build
```
If I try now to run `make html`

I get these output :) 

![Screenshot from 2019-09-01 14-54-22](https://user-images.githubusercontent.com/358860/64076709-6d40ce80-ccc8-11e9-934d-94e052df55dc.png)

(Click on the pic for a larger view)

OS: 
```
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: Fedora
Description:    Fedora release 30 (Thirty)
Release:        30
Codename:       Thirty
```
Python: 3.7

**Comment by svx** [comment]
Update:
Doing the same steps, this time on Ubuntu I get the same error :)

```shell
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.04
Release:	19.04
Codename:	disco
```

**Comment by vangheem** [comment]
Ahh yes, looks like doc builds have been failing since we merged latest swagger/open api changes: https://readthedocs.org/projects/guillotina/builds/9590317/

**Comment by svx** [comment]
Fixed by #661, closing this.
